src/features/update/update-model.spec.ts

```diff
@@ -416,8 +416,20 @@ describe('UpdateModelCommand', () => {
         }
     });
 
-    // tests regression #190
-    it('removes relocated elements before adding them', () => {
+    it('#190 removes relocated elements before adding them { animate: false }', async () => {
+        removesRelocatedElementsBeforeAddingThem(false);
```

Comment:
Isn't this missing an `await`(or simply `return`, then you don't need `async`)?
Same for other invocations below.

Reply:
Good catch! The return is missing.